cacheops/redis.py

```diff
@@ -29,24 +30,48 @@ def handle_connection_failure(call):
 
 client_class_name = getattr(settings, 'CACHEOPS_CLIENT_CLASS', None)
 client_class = import_string(client_class_name) if client_class_name else redis.StrictRedis
+redis_replicas = []
+
+
+def ip(url):
+    return socket.gethostbyname(urlparse(url).hostname)
+
+
+def set_redis_replicas():
```

Comment:
Maybe `rebalance_replicas`?  I think that's what's going on here, and that's the word you used when you walked me through the logic a couple of minutes ago.